packages/web-react/src/components/Stack/useStackStyleProps.ts

```diff
@@ -1,17 +1,23 @@
 import classNames from 'classnames';
-import { ElementType } from 'react';
+import { CSSProperties, ElementType } from 'react';
 import { useClassNamePrefix } from '../../hooks';
 import { SpiritStackProps } from '../../types';
 
+interface StackCSSProperties extends CSSProperties {
```

Comment:
I think I have seen similar type somewhere in our code. 🤔

Reply:
similar code is in griditem, but I am not sure how to correctly abstract it. (name and placement)

Reply:
Btw, please look at https://github.com/frenic/csstype#what-should-i-do-when-i-get-type-errors. I think that the point 3 could help you to achieve what you want.